lib/rubygems/specification.rb

```diff
@@ -112,6 +112,31 @@ class Gem::Specification < Gem::BasicSpecification
 
   VALID_NAME_PATTERN = /\A[a-zA-Z0-9\.\-\_]+\z/ # :nodoc:
 
+  DEFAULT_GEMS_LIST = %w[
```

Comment:
shouldn't this pull from `Gem::Specification.select(&:default_gem?).map(&:name)` ?

Reply:
It's not working. default gems were determined without ruby version.

Reply:
>  default gems were determined without ruby version

I'm not sure I understand what you mean by this?

Reply:
Default gems do not depend on Ruby version.

Ex. We can install default gems from listed by https://stdgems.org/  with old ruby. But is not activated in the current specification. I hope to activate them with old ruby.